### PR TITLE
Deprecation note

### PR DESCRIPTION
**Pull request by fabianaferreira**
Adding a note on the first step letting students know that this course is currently on another portal.

- Deletes all steps except for the first
- Adds a note on the first step
- The template remains the same
- `config.json` file was changed to have only one step

### CODE REVIEW

**Review by gris — APPROVED** [review]

File: course/steps/01_manifest/en.md [hunk 1]
```diff
@@ -1,67 +1,7 @@
...
-  },
-  "dependencies": {
-    "vtex.styleguide": "9.x",
-    "vtex.css-handles": "0.x"
-  },
-  "$schema": "https://raw.githubusercontent.com/vtex/node-vtex-a pi/master/gen/manifest.schema"
-}
-```
-
+> **This content was moved to another place. Soon we'll no longer support these courses and learning portal**
```
Comment (gris) [inline course/steps/01_manifest/en.md:3]: Talvez colocar que não suportamos mais aqui ja

File: course/steps/01_manifest/pt.md [hunk 2]
```diff
@@ -1,72 +1,7 @@
 # Conhecendo uma *app* VTEX IO
 
-## Introdução
-Antes de começar, é necessário relembrar alguns conceitos importantes para uma maior compreensão do fluxo lógico ao desenvolver uma *app*.
+> **Este conteúdo foi movido para um outro lugar. Em breve, não daremos mais suporte a estes cursos e a este canal de aprendizado**
```
Comment (gris) [inline course/steps/01_manifest/pt.md:3]: Mesma coisa aqui

**Comment by gris** [comment]
https://media.giphy.com/media/5PSPV1ucLX31u/giphy.gif

**Review by gris — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Added deprecation note
```
Comment (lucis) [inline CHANGELOG.md:10]: Just as a tip: I usually hear more the term `Deprecation Notice` _(years of PHP taught me that)_

**Review by lucis — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (FabianaFerreira) [reply CHANGELOG.md:10]: Nice one! Thank you for letting me know! 🚀

**Pull request merged**